--- ./ricty_generator-4.1.1.sh	2024-09-15 11:16:31
+++ ./ricty_generator-4.1.1-after.sh	2024-09-15 11:46:23
@@ -1097,12 +1097,6 @@
 $fontforge_command -script ${tmpdir}/${modified_migu1m_generator} \
     2> $redirection_stderr || exit 4
 $fontforge_command -script ${tmpdir}/${ricty_generator} \
-    2> $redirection_stderr || exit 4
-$fontforge_command -script ${tmpdir}/${ricty_discord_generator} \
-    ${ricty_familyname}${ricty_familyname_suffix}-Regular.ttf "${non_discorded_characters}" \
-    2> $redirection_stderr || exit 4
-$fontforge_command -script ${tmpdir}/${ricty_discord_generator} \
-    ${ricty_familyname}${ricty_familyname_suffix}-Bold.ttf "${non_discorded_characters}" \
     2> $redirection_stderr || exit 4
 $fontforge_command -script ${tmpdir}/${regular2oblique_converter} \
     ${ricty_familyname}${ricty_familyname_suffix}-Regular.ttf \
@@ -1110,12 +1104,6 @@
 $fontforge_command -script ${tmpdir}/${regular2oblique_converter} \
     ${ricty_familyname}${ricty_familyname_suffix}-Bold.ttf \
     2> $redirection_stderr || exit 4
-$fontforge_command -script ${tmpdir}/${regular2oblique_converter} \
-    ${ricty_familyname}${ricty_familyname_suffix}Discord-Regular.ttf \
-    2> $redirection_stderr || exit 4
-$fontforge_command -script ${tmpdir}/${regular2oblique_converter} \
-    ${ricty_familyname}${ricty_familyname_suffix}Discord-Bold.ttf \
-    2> $redirection_stderr || exit 4
 
 # Remove temporary directory
 if [ "${leaving_tmp_flag}" = "false" ]
